src/commands/build.ts

```diff
@@ -86,8 +89,47 @@ async function build_action(options: BuildOptions, context_file: string): Promis
       }
       return stdout.replace(/^\s+|\s+$/g, '');
     }
-  }, async (options) => {
-    return await command_helper.componentStore.addVolume(options.host_path);
+  }, async (build_options) => {
+    console.log('Building image for volume', build_options.deployment_name + '.volumes.' + build_options.volume_name);
+
+    // Create the directory for the new volume container
+    const tmpDir = await Deno.makeTempDir();
+    await mod.copy(build_options.host_path, path.join(tmpDir, 'contents'));
+    Deno.writeTextFileSync(
+      path.join(tmpDir, 'Dockerfile'),
+      `
+        FROM alpine:latest
+        WORKDIR /app
+        COPY ./contents .
+        CMD ["sh", "-c", "cp -r ./* $TARGET_DIR"]
+      `,
+    );
+
+    // Publish the volume container
+    if (options.verbose) {
```

Comment:
Could a lot of this code be merged into one path? The code above is pretty streamlined where the only difference is that the verbose flag decides on the quiet flag.

Reply:
Consolidated into a helper function.